Sources/Playbook/Components/Popover/Popover+Position.swift

```diff
@@ -53,22 +53,22 @@ public extension Position {
       let safeWindowFrame = Screen.rect
       let maxX = safeWindowFrame.maxX - screenEdgePadding.trailing
       let maxY = safeWindowFrame.maxY - screenEdgePadding.bottom
-      #if os(macOS)
-      if popoverFrame.origin.x < screenEdgePadding.leading {
-        popoverFrame.origin.x = screenEdgePadding.leading
-      }
-      if popoverFrame.origin.y < screenEdgePadding.top {
-        popoverFrame.origin.y = screenEdgePadding.top
-      }
-      if popoverFrame.maxX > maxX {
-        let difference = popoverFrame.maxX - maxX
-        popoverFrame.origin.x -= difference
-      }
-      if popoverFrame.maxY > maxY {
-        let difference = popoverFrame.maxY - maxY
-        popoverFrame.origin.y -= difference
-      }
-      #endif
+//      #if os(macOS)
+//      if popoverFrame.origin.x < screenEdgePadding.leading {
+//        popoverFrame.origin.x = screenEdgePadding.leading
+//      }
+//      if popoverFrame.origin.y < screenEdgePadding.top {
+//        popoverFrame.origin.y = screenEdgePadding.top
+//      }
+//      if popoverFrame.maxX > maxX {
+//        let difference = popoverFrame.maxX - maxX
+//        popoverFrame.origin.x -= difference
+//      }
+//      if popoverFrame.maxY > maxY {
+//        let difference = popoverFrame.maxY - maxY
+//        popoverFrame.origin.y -= difference
+//      }
+//      #endif
```

Comment:
Is it still needed?

Reply:
yes. We will need this to fix the margins. It's a future improvement.